tests/testthat/test-parser.R

```diff
@@ -5,6 +5,7 @@ test_that("Correct git repo path", {
 })
 test_that("Calling parse_gitlog with correct perceval and correct git log path returns a data table", {
   tools_path <- "../../tools.yml"
+  tools_path <- file.path(getwd(), tools_path)
```

Comment:
Hi Malia,

`getwd()` will not necessarily work. If the user changes their directory via setwd() this will fail too.  I am curious on why you added this too. Was `filepath(tools_path)` not working for you?

Make sure you

Reply:
It works, I'm just wondering if the errors happen because it didn't found the perceval path

Reply:
Just to be clear: Does it work without `getwd()`?

Reply:
Yes

Reply:
You are correct in that the test will fail via the Check for not finding Perceval. I am just trying to understand if we need the "getwd()" here.

Reply:
We don't need getwd()